Heart/User/Domain/Entities/InformationEntity.php

```diff
@@ -5,17 +5,27 @@
 class InformationEntity implements \JsonSerializable
 {
     public function __construct(
-        private readonly string $id,
-        private readonly string $userId,
-        private readonly string $name,
-        private readonly string $nickname,
-        private readonly string $linkedinUrl,
-        private readonly string $githubUrl,
-        private readonly ?string $birthdate,
-        private readonly string $about,
+        private string $id,
+        private string $userId,
+        private ?string $name,
+        private ?string $nickname,
+        private ?string $linkedinUrl,
+        private ?string $githubUrl,
+        private ?string $birthdate,
+        private ?string $about,
```

Comment:
If your goal was to make this properties nullables, you should also define the fallback value as = null